### ISSUE TITLE
Wrong error when creating a docker file that is using a base image from an argument

### ISSUE THREAD

**Issue by baruchoxman** [replicatedhq/dockerfilelint#197]
I have created a docker file that gets the base image name from a variable:

```
ARG BASE_IMAGE=my-image:base

# Use the base image
FROM $BASE_IMAGE
...
```

And I get the following error:
```
Issues: 1

Line 4: FROM $BASE_IMAGE
Issue  Category      Title                 Description
    1  Clarity       Base Image Missing    Base images should specify a tag to use.
                     Tag
```

**Comment by make-github-pseudonymous-again** [comment]
This is a duplicate of:
- #100

Which was fixed with:
- #151

However there has not been a new release since #151 was merged which is why the bug can still be reproduced. See:
- #192
